docs/installation/my_token.md

```diff
@@ -6,7 +6,7 @@ It contain 3 words as :
 This is the user whose branch of that repository is going to be trigger.
 We are using [fossasia/meilx](https://github.com/fossasia/meilix) repository to trigger its build.
 
-**Before undergoing any development process you must fork [fossasia/meilx](https://github.com/fossasia/meilix) repository and change the username so that the release will be done in your repository**
+**Before undergoing any development process you must fork [fossasia/meilx](https://github.com/fossasia/meilix) repository and change the username so that the release will be done in your repository**.
 You can fork [this](https://github.com/fossasia/meilix) and start to put **your username** so that the fossasia/meilix repo will not get flooded with unnecessary builds. And your can easily play with configuration without disturbing the original repo.
```

Comment:
It would be great if we specify the file in which we have to change it. So that there is no confusion.